pkg/features/features.go

```diff
@@ -86,6 +86,11 @@ const (
 	// owner: @docktofuture
 	// beta: v1.133.0
 	CustomDNSServerInNodeLocalDNS featuregate.Feature = "CustomDNSServerInNodeLocalDNS"
+
+	// VpnBondingModeRoundRobin enables the usage of the "balance-rr" bonding mode for the HA VPN setup.
+	// owner: @domdom82
+	// beta: v1.133.0
```

Comment:
[v1.134.0](https://github.com/gardener/gardener/releases/tag/v1.134.0) was released yesterday. New content in the master branch will be part of v1.135.0.

```suggestion
	// beta: v1.135.0
```